apps/website/src/components/content/YouTube.tsx

```diff
@@ -146,206 +146,207 @@ type LightboxProps = {
+      | string
+      | undefined;
+    if (active === value) return;
+
+    // Get the gallery and children
+    const gallery = resolvePhotoswipeElementProvider(
+      photoswipe.options.gallery,
+    )?.[0];
+    if (!gallery) return;
+    const items = resolvePhotoswipeElementProvider(
+      photoswipe.options.children,
+      gallery,
     );
-
-    // Allow children to be functions that receive the context
-    const childrenToRender = useMemo(
-      () =>
-        (Array.isArray(children) ? children : [children]).map(
-          (child, index) => {
-            const element = typeof child === "function" ? child(ctx) : child;
-            return cloneElement(element, { key: index });
-          },
-        ),
-      [children, ctx],
+    if (!items) return;
+
+    // Locate the matching trigger and open it
+    // If we can't find it, set the value back to the active slide
+    const match = items.findIndex(
+      (child) =>
+        safeJsonParse(child.getAttribute(`data-lightbox-${photoswipeId}`) || "")
+          ?.triggerId === value,
     );
+    if (match !== -1)
+      photoswipe.loadAndOpen(
+        match,
+        photoswipe.options.dataSource || { gallery, items },
+      );
+    if (match === -1 && onChange) onChange(active);
+  }, [value, photoswipe, photoswipeId, onChange]);
+
+  // Expose the nested components
+  const ctx: LightboxCtxProps = useMemo(
+    () => ({
+      Trigger: createTrigger(photoswipeId),
+      parseUrl,
+      id: photoswipeId,
+    }),
+    [photoswipeId],
+  );
 
-    return (
-      <>
-        <style
-          dangerouslySetInnerHTML={{
-            // Hide the placeholder overlay
-            // Ensure we retain the aspect ratio of the video
-            __html: `
+  // Allow children to be functions that receive the context
+  const childrenToRender = useMemo(
+    () =>
+      (Array.isArray(children) ? children : [children]).map((child, index) => {
+        const element = typeof child === "function" ? child(ctx) : child;
+        return cloneElement(element, { key: index });
+      }),
+    [children, ctx],
+  );
+
+  return (
+    <>
+      <style
+        dangerouslySetInnerHTML={{
+          // Hide the placeholder overlay
+          // Ensure we retain the aspect ratio of the video
+          __html: `
 .pswp--${photoswipeId} .pswp__img--placeholder {
-  display: none;
+display: none;
 }
 
 .pswp--${photoswipeId} .pswp__item[aria-hidden="false"] .pswp--${photoswipeId}-wrapper {
-  container-type: size;
-  container-name: photoswipe-${photoswipeId};
+container-type: size;
+container-name: photoswipe-${photoswipeId};
 }
 
 @container photoswipe-${photoswipeId} (aspect-ratio > 16/9) {
-  .pswp--${photoswipeId} .pswp__item[aria-hidden="false"] .pswp--${photoswipeId}-wrapper iframe {
-    width: auto;
-    height: 100%;
-  }
+.pswp--${photoswipeId} .pswp__item[aria-hidden="false"] .pswp--${photoswipeId}-wrapper iframe {
+  width: auto;
+  height: 100%;
+}
 }
```

Comment:
Can we make prettier (?) not do that?

Reply:
My bad, not prettier's, when I de-dented the whole component